test/molecule/smartSlider/index.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Remember: YOUR COMPONENT IS DEFINED GLOBALLY
+ * */
+
+/* eslint react/jsx-no-undef:0 */
+/* eslint no-undef:0 */
+
+import ReactDOM from 'react-dom'
+
+import chai, {expect} from 'chai'
+import chaiDOM from 'chai-dom'
+
+chai.use(chaiDOM)
+
+describe('MoleculeSmartSlider', () => {
+  const Component = MoleculeSmartSlider
+  const setup = setupEnvironment(Component)
+
+  it('should render without crashing', () => {
+    // Given
+    const props = {}
+
+    // When
+    const component = <Component {...props} />
+
+    // Then
+    const div = document.createElement('div')
+    ReactDOM.render(component, div)
+    ReactDOM.unmountComponentAtNode(div)
+  })
+
+  it('should NOT render null', () => {
+    // Given
+    const props = {}
+
+    // When
+    const {container} = setup(props)
+
+    // Then
+    expect(container.innerHTML).to.be.a('string')
+    expect(container.innerHTML).to.not.have.lengthOf(0)
+  })
+
+  it('example to be deleted', () => {
+    // Example TO BE DELETED!!!!
+
+    // Given
+    // const props = {}
+
+    // When
+    // const {getByRole} = setup(props)
+
+    // Then
+    // expect(getByRole('button')).to.have.text('HOLA')
+    expect(true).to.be.eql(false)
```

Comment:
Tests are not going through. I'm 100% sure 😂

Reply:
ahha you're right... forget about it... it's just a draft / POC